gulpfile.js

```diff
@@ -102,9 +110,25 @@ gulp.task("js:vendors", function() {
 /**
  * Server task
  */
+// gulp.task("server", function() {
+//    return gulp.src(opt.outputFolder)
+//     .pipe(webserver(opt.server));
+// });
+//
+var connect = require('connect');
+var livereload = require('connect-livereload');
+var staticFile = require('serve-static');
+var lrServer = require('tiny-lr')();
```

Comment:
Last time I use it, gulp-webserver was laggy that's why I change to a that solution